rholang/src/test/scala/coop/rchain/rholang/StackSafetySpec.scala

```diff
@@ -71,11 +90,17 @@ class StackSafetySpec extends FlatSpec with Matchers {
   }
 
   it should "handle a huge nested list" in {
-    checkNormalize(hugeNested("[", "", "]"))
+    checkAll(hugeNested("[", "", "]"))
   }
 
+  //FIXME java.lang.OutOfMemoryError: GC overhead limit exceeded
+  //it should "handle a huge nested set" in {
+  //  checkAll(hugeNested("Set(", "42", ")"))
+  //}
+
+  //FIXME 11.5 s
   it should "handle a nested new" in {
-    checkNormalize(hugeNested("new x in { ", "1", "}"))
+    checkAll(hugeNested("new x in { ", "1", "}"))
```

Comment:
This took 23 seconds on my machine.
Just wondering if it shouldn't actually fail because of `val maxDuration = 10.seconds`?

Reply:
`checkAll` consists of: `buildNormalizedTerm`, pretty-printing, and doing a full reduce from rholang sources (which includes buildNormalizedTerm again). Only the last part is under the timeout.

Reply:
I've just seen 26 s on that, so it's not just your machine - it might be there's been some overhead introduced by the stacksafety. We won't fight it for now though.